brouter-expressions/src/main/java/btools/expressions/IntegrityCheckProfile.java

```diff
@@ -0,0 +1,50 @@
+package btools.expressions;
```

Comment:
Why is it necessary to add a commandline util? Which tests can't be handled using normal tests?

Reply:
The command line tool is for real life.
JUnit works at the end of the development process.

Reply:
I think our workflows are quite different. When using Test-driven development tests are written before the implementation and help while developing. Is the command line tool used after implementation is complete?

Reply:
Our views are different to testing.
my: 5+5 = 10
your: 10 = 5+5
Both have the result 10. But what is when you start with the result 
10 and get the way 6+4. Result is ok but the way could be wrong.
And what is when the best way is 5+6. You will never find it.
Other sample:
49 Euro ticket, the result is clear but the ways to get there? Or is a 1 Euro per day ticket a better solution? (Sorry for the German sample)
More brouter sample:
We had the PR #474 with test server - pretty good idea to test that way.
Let's take a look at the 'overrideParameter' routine. The result is 1902 for track length. Why that, why not 42 like every on question? You have previously routed to get this value. If we work on the new library now - some day - this test will break. Because of different length. But the intent is still perfect: the override parameter still works. Only the test is wrong.

My workflow is to test every line I write - on the perfect way. But this means for me:
- 1. line has its result in a log file
- 2. line in gpx file
- 3. line needs a diff on two gpx files
- 4. line is only visible on a graphical output test
So I think this is also test driven.
In a pure form and not in the restriction of junit.

Reply:
I honestly don't understand your references, but I think there is enough information regarding test driven development available.

I actually think that my tests in #474 are bad tests because they check the route length, but that's the only thing that can be observed. Because of the architecture and missing dependcy injection it's impossible to write real unit tests instead of system tests.